examples/twr_aloha/Makefile

```diff
@@ -44,4 +44,7 @@ ifneq (,$(filter llvm,$(TOOLCHAIN)))
   CFLAGS += -Wno-microsoft-anon-tag
 endif
 
+# The test only works on dwm1001 BOARDs, there is none on CI
+TEST_ON_CI_BLACKLIST = all
```

Comment:
Because that is not the, case you could connect another dw1000 module and it "should" work.

Reply:
Hmm maybe change to `test-with-config`?

Reply:
>  Hmm maybe change to test-with-config?

Why ? IMHO, it's ok as is since it seems that no preliminary configuration is required (assuming installing riot_ctrl is not a configuration).

Reply:
> Why ? IMHO, it's ok as is since it seems that no preliminary configuration is required (assuming installing riot_ctrl is not a configuration).

Well the thing is that the test would be ran on other BOARDs that do not have the `dw1000` module, e.g. the same `iotlab-` action would test it, since `TEST_ON_CI_WHITELIST` is a murdock specific thing AFAIK. I could also use `BOARD_WHITELIST` but that would prevent building the application, which I don't want.

Reply:
> > Why ? IMHO, it's ok as is since it seems that no preliminary configuration is required (assuming installing riot_ctrl is not a configuration).
> 
> Well the thing is that the test would be ran on other BOARDs that do not have the `dw1000` module, e.g. the same `iotlab-` action would test it, since `TEST_ON_CI_WHITELIST` is a murdock specific thing AFAIK. I could also use `BOARD_WHITELIST` but that would prevent building the application, which I don't want.

I switched the test to `test-with-config`.